tfe.go

```diff
@@ -870,8 +872,8 @@ func decodeErrorPayload(r *http.Response) ([]string, error) {
 	return errs, nil
 }
 
-func errorPayloadContains(errors []string, match string) bool {
-	for _, e := range errors {
+func errorPayloadContains(payloadErrors []string, match string) bool {
```

Comment:
unrelated change to this PR, fyi. Golangci-lint warned me that the previous name assigned to this variable was clashing with the name of the external library "errors".